iop/kernel/include/iomanX.h

```diff
@@ -87,6 +87,11 @@ typedef struct _iomanX_iop_device {
 	struct _iomanX_iop_device_ops *ops;
 } iomanX_iop_device_t;
 
+
+#define DECL_NOT_SUPPORTED_OP() \
```

Comment:
instead of using `macro`s you can also use:
```c
static inline int not_supported_op (void) {return -134;}
```

Reply:
You're right, I'll apply the change later.

Although, I would like to read a review of @uyjulian on this regard, specially with the DVR related IRX modules